File: math/fizzbuzz.rb
# @param {Integer} n
# @return {String[]}
def fizz_buzz(n)
    res = []
    (1..n).each do |num|
        if num % 5 == 0 && num % 3 == 0
            res << "FizzBuzz"
        elsif num % 5 == 0
            res << "Buzz"
        elsif num % 3 == 0
            res << "Fizz"
        else
            res << num.to_s
        end
    end

    res
end

File: math/fizzbuzz.py
class Solution:
    def fizzBuzz(self, n: int) -> List[str]:
        res = []
        for num in range(1, n + 1):
            if num % 5 == 0 and num % 3 == 0:
                res.append("FizzBuzz")
            elif num % 5 == 0:
                res.append("Buzz")
            elif num % 3 == 0:
                res.append("Fizz")
            else:
                res.append(num.to_s)
            

        return res